src/luaharfbuzz/face.c

```diff
@@ -596,6 +596,27 @@ static int face_var_normalize_coords(lua_State *L) {
   return count;
 }
 
+// face_blob(face) -> harfbuzz.Blob|nil, err
+// Returns the underlying face blob as a harfbuzz.Blob userdata.
+static int face_blob(lua_State *L) {
+  Face *f = (Face *)luaL_checkudata(L, 1, "harfbuzz.Face");
+
+  hb_blob_t *hb = hb_face_reference_blob(*f);
+  if (!hb) {
+    lua_pushnil(L);
+    lua_pushliteral(L, "face has no backing blob");
```

Comment:
I don’t think this is needed. The caller should check the return value and decide what do do. Such messages are hard to suppress.